src/test/java/edu/ucsb/cs156/example/services/UCSBSubjectsServiceTests.java

```diff
@@ -1,57 +1,91 @@
 package edu.ucsb.cs156.example.services;
 
+import edu.ucsb.cs156.example.entities.UCSBSubject;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.header;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
+import org.springframework.test.web.client.MockRestServiceServer;
```

Comment:
small nitpick: can we keep all the `org.springframework` imports together and alphabetized?

Reply:
will do from now onwards!